lib/mapper.js

```diff
@@ -52,6 +49,7 @@ exports.page = function(page, settings){
 
 exports.screen = function(screen, settings){
   var result = createCommonGAForm(screen, settings);
```

Comment:
@RyanDulon - unsure why're you're being copied. There's an unsubscribe button near the top - https://cloudup.com/ckQda09NU1z